UserTaskSample/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (C) 2023 by Genetec, Inc. All rights reserved.
+// May be used only in accordance with a valid Source Code License Agreement.
+
+namespace Genetec.Dap.CodeSamples
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Genetec.Sdk.Entities;
+    using Sdk;
+    using Sdk.UserTaskManagement;
+
+    class Program
+    {
+        static Program() => SdkResolver.Initialize();
+
+        static async Task Main()
+        {
+            const string server = "localhost";
+            const string username = "admin";
+            const string password = "";
+
+            using var engine = new Engine();
+
+            ConnectionStateCode state = await engine.LogOnAsync(server: server, username: username, password: password);
+
+            if (state == ConnectionStateCode.Success)
+            {
+                IEnumerable<PrivateUserTask> privateTasks = engine.UserTaskManager.GetPrivateTasks().OfType<PrivateUserTask>().ToList();
```

Comment:
Think we could add a comment here that GetPrivateTasks returns the private tasks for the current user, and overrides of the GetPrivateTasks function can get the tasks of a specified user, in order to avoid any confusion that GetPrivateTasks may return all private tasks. The documentation specifies this but the name of the function alone within the sample may mislead some developers.

Reply:
Thanks for the suggestion. The existing documentation already clarifies that GetPrivateTasks returns the private tasks for the current user and that overrides can be used to get the tasks of a specified user. Given this, adding a comment in the code may be redundant. Developers should refer to the documentation for detailed information on the function's behavior.